src/bin/pgcopydb/copydb.c

```diff
@@ -709,14 +709,9 @@ copydb_init_table_specs(CopyTableDataSpec *tableSpecs,
 		.cfPaths = &(specs->cfPaths),
 		.pgPaths = &(specs->pgPaths),
 
-		.source_pguri = { 0 },
-		.target_pguri = { 0 },
-		.sourceSnapshot = {
-			.pgsql = { 0 },
-			.pguri = { 0 },
-			.connectionType = specs->sourceSnapshot.connectionType,
-			.snapshot = { 0 }
-		},
```

Comment:
Is is safe to share the same snapshot as in the main specs? if that's safe, is there a good reason to keep a snapshot instance in the CopyTableDataSpec at all?

Reply:
`sourceSnapshot` isn't providing any value so I removed it.